### PR TITLE
Sort sound library files by ascii code

### PR DESCRIPTION
**Pull request by ndr-brt**
As discussed in #131

### CODE REVIEW

**Comment by telephon** [comment]
How do we handle breaking changes @yaxu ?

**Comment by yaxu** [comment]
@telephon In tidal I release a major version (according to https://pvp.haskell.org/ ) and put some details in a CHANGELOG file

However with the discussion around ordering from metadata files, maybe it isn't worth the breaking change here?

**Comment by yaxu** [comment]
I think this isn't a breaking change under linux btw. I think the difference between platforms is whether the extension is considered in the sorting.

**Comment by telephon** [comment]
How should we proceed here? I think it is good to have a consistent ordering. 

On macOS there seems to be no difference, here is a test:
```supercollider

(
var k;
k = { |e| e[\bufferObject].path.basename };

~dirt.soundLibrary.bufferEvents.select { |x|
	x.order { |a, b| k.(a) < k.(b) } != (0..x.size-1)
};
)

```

**Comment by telephon** [comment]
Good!  I had forgotten about your PR. Making it globally configurable seems like a bad idea in my version. Probably better as an argument to the function itself. Would you like to add this to yours?

```
loadSoundFileFolder { |folderPath, name, appendToExisting = false, sortFiles = true|

…
if(sortFiles) { files.sort };
…
```

**Comment by ndr-brt** [comment]
@telephon done! added the argument also to the outer `loadSoundFileFolder`

File: classes/SuperDirt.sc [hunk 1]
```diff
@@ -124,7 +124,7 @@ SuperDirt {
 		soundLibrary.loadOnly(names, path, appendToExisting )
 	}
 
-	loadSoundFileFolder { |folderPath, name, appendToExisting = false|
+	loadSoundFileFolder { |folderPath, name, appendToExisting = false, sortFiles = true|
 		soundLibrary.loadSoundFileFolder(folderPath, name, appendToExisting)
```
Comment (telephon) [inline classes/SuperDirt.sc]: you have to pass the argument on to `soundLibrary.loadSoundFileFolder` …

**Review by telephon — CHANGES_REQUESTED** [review]
just one little thing …

File: classes/SuperDirt.sc (hunk 1 above)
Comment (ndr-brt) [reply classes/SuperDirt.sc]: Yes, you're right, added it.

**Review by telephon — APPROVED** [review]

**Pull request merged**